pkg/migrations/sql/idempotent/base.sql

```diff
@@ -1952,13 +2011,20 @@ BEGIN
         END IF;
 END
 $func$
-LANGUAGE PLPGSQL STABLE;
+LANGUAGE PLPGSQL STABLE
+SECURITY DEFINER
+--search path must be set for security definer
+--need to include public(the timescaledb schema) for some timescale functions to work.
+SET search_path = public, pg_temp;
```

Comment:
The timescale functions don't schema qualify so we need public in the search path? this seems like a bug that we should report upstream?